src/config.js

```diff
@@ -9,7 +9,7 @@ module.exports.RATE_DELAY = 15 * 60 * 1000;
 
 module.exports.LND_PORT = 10009;
 module.exports.LND_PEER_PORT = 10019;
-module.exports.MACAROONS_ENABLED = false;
+module.exports.NETWORK = 'testnet';
```

Comment:
What if we're in dev mode? Then the app will start with `testnet` instead of `simnet`?